### PR TITLE
Featured/closed

### PR DESCRIPTION
**Pull request by buljagung**

### CODE REVIEW

File: apps/ufjtbot/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "lint": "eslint . --fix",
     "start": "node --experimental-json-modules build/main.js",
+    "watch": "pnpm build -- -w",
     "build": "swc src -d build -C jsc.parser.syntax=typescript -C module.type=es6 -C jsc.target=es2021 -C jsc.externalHelpers=true -C jsc.experimental.keepImportAssertions=true --no-swcrc",
     "docker:build": "cmd dockerized apps/ufjtbot/Dockerfile --exclude=__tests__/** .env.example | docker build --build-arg PACKAGE_PATH=apps/ufjtbot - -t mono-bot"
```
Comment (hasft) [inline apps/ufjtbot/package.json:11]: lelelele

File: apps/ufjtbot/src/github/pullRequest/closed.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { Context } from 'probot';
 import { logger } from '@ufjt-poc/logger';
 import { getErrorMessage, useSlackClient } from '../../utils.js';
-import { getChannelsFromRepository, getPullRequestMessages } from '../../requests.js';
+import { getChannelsFromRepository, getPullRequestMessages, removeConversation } from '../../requests.js';
 import mergedMessage from '../../slack/blocks/mergedMessage.js';
 
 export default async function closed({ payload }: Context<'pull_request.closed'>) {
```
Comment (hasft) [inline apps/ufjtbot/src/github/pullRequest/closed.ts:7]: oooooo

**Review by hasft — COMMENTED** [review]
coba

File: apps/ufjtbot/package.json (hunk 1 above)
Comment (hasft) [inline apps/ufjtbot/package.json:11]: lalalala

File: apps/ufjtbot/package.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "lint": "eslint . --fix",
     "start": "node --experimental-json-modules build/main.js",
+    "watch": "pnpm build -- -w",
     "build": "swc src -d build -C jsc.parser.syntax=typescript -C module.type=es6 -C jsc.target=es2021 -C jsc.externalHelpers=true -C jsc.experimental.keepImportAssertions=true --no-swcrc",
```
Comment (buljagung) [inline apps/ufjtbot/package.json:10]: dododod

Comment (buljagung) [reply apps/ufjtbot/package.json:10]: ddada

Comment (buljagung) [reply apps/ufjtbot/package.json:10]: alalaa

Comment (buljagung) [reply apps/ufjtbot/package.json:10]: kokokok

Comment (buljagung) [reply apps/ufjtbot/package.json:10]: obla

**Pull request closed without merging**